proxy/src/main/java/io/openlineage/proxy/api/models/ConsoleLineageStream.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.openlineage.proxy.api.models;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+/**
+ * ConsoleLineageStream pushes events to stdout
+ */
+public class ConsoleLineageStream extends LineageStream
+{
+  public ConsoleLineageStream()
+  {
+    super(Type.CONSOLE);
+  }
+
+  @Override
+  public void collect(String event)
+  {
+    System.out.println(event);
```

Comment:
_Minor_: Should we use the `logger` here (`log.info()`), instead of `println()`?